.github/workflows/schedule-daily.yml

```diff
@@ -136,10 +136,11 @@ jobs:
           BAZEL_COMMAND: "test"
           BAZEL_TARGETS: "//rs/rosetta-api/..."
           BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
-          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=fi_tests_nightly --test_env=SSH_AUTH_SOCK --test_timeout=7200"
+          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=fi_tests_nightly --test_env=SSH_AUTH_SOCK"
           HONEYCOMB_API_TOKEN: ${{ secrets.HONEYCOMB_API_TOKEN }}
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
       - name: Upload bazel-bep
+        if: always()
```

Comment:
I think this will break the workflow, there's two `if`s

Reply:
Oh shoot, it looks like this was added by the IDX GitHub Automation. I'll see if first merging the latest master helps.

Reply:
Good catch, @nmattia! I only looked at the rosetta-related changes. The `--test_timeout` that was removed also needs to stay, otherwise the recently-added FI nightly tests will time out.